loader/src/injector/hook.cpp

```diff
@@ -592,13 +594,13 @@ void ZygiskContext::fork_pre() {
 
     /* INFO: Record all open fds */
     DIR *dir = opendir("/proc/self/fd");
-    if (dir == nullptr) {
-        PLOGE("Failed to open /proc/self/fd");
+    if (dir == NULL) {
+        PLOGE("Failed to open %s", "/proc/self/fd");
 
         return;
     }
 
-    struct dirent *entry;
+    struct dirent const *entry;
```

Comment:
Any specific reason? Aside from some optimization? Sometimes it's better to leave constless honestly and let the compiler figure out